app/pages/home.cjsx

```diff
@@ -10,6 +9,11 @@ ProjectCard = require '../partials/project-card'
 alert = require '../lib/alert'
 LoginDialog = require '../partials/login-dialog'
 
+FEATURED_PRODUCT_IDS = if process.env.NODE_ENV is 'production'
```

Comment:
Why not expose the env from `app/api/config.coffee` and use whatever comes out of that?